### ISSUE TITLE
Notifications for delayed entry

### ISSUE THREAD

**Issue by cantchooseaname8** [dgreif/ring-alarm#20]
I’m not sure if this is supposed to work or not, but I don’t get any notifications for delayed entry. I only get notifications when it’s armed and disarmed. Is there any way to set it up to get notifications when the alarm goes into delayed entry? I’ve had problems where a motion sensor trips the delayed entry but I don’t hear or realize it’s going off until the alarm siren triggers. These notifications can be set up in the Ring app, but would be so much more useful if they came through HomeKit. Having a notification would be a huge help.

**Comment by dgreif** [comment]
@cantchooseaname8 thanks for the request.  I'm not sure that everyone would want notifications for every entry delay, since that would mean a notification every time they entered their home and the alarm was on away mode.  I can definitely see the upside though, and I will look into adding a configuration option that would allow you to choose if entry delay sets HomeKit to "Alarming".

**Comment by cantchooseaname8** [comment]
@dgreif thanks so much for the quick reply. A configurable option to have notifications when entry delay is triggered either from away or home mode would be amazing. Thanks so much for all your work on this plugin and sharing it with everyone!

**Comment by dgreif** [comment]
@cantchooseaname8 I added an `alarmOnEntryDelay` option in `v1.5.0`.  This will actually work really well for me as well since I have mine set up for auto arm/disarm on leave/arrive.  I've had a lot of instances of visitors setting off the alarm when I'm not home.  Thanks again for the request!

**Comment by cantchooseaname8** [comment]
@dgreif wow that was fast! I’ll take a look and test this out tomorrow. Thank you so much!

**Comment by cantchooseaname8** [comment]
I was just able to test it out. The notifications popped up perfectly. The status in HomeKit showed “Triggered” and this is where I ran into a problem. When I went to change the status to “Off”, it was already highlighted and wouldn’t let me turn the alarm off. I had to use the keypad. Is that how it is supposed to work in HomeKit? Maybe a triggered alarm can’t be turned off in HomeKit? Just want to make sure I didn’t mess something up. Thanks!
